editor/shader_create_dialog.cpp

```diff
@@ -164,37 +164,76 @@ void ShaderCreateDialog::_create_new() {
 			code += vformat("shader_type %s;\n", mode_menu->get_text().to_snake_case());
 
 			if (current_template == 0) { // Default template.
-				code += "\n";
 				switch (current_mode) {
 					case Shader::MODE_SPATIAL:
-						code += "void fragment() {\n";
-						code += "\t// Place fragment code here.\n";
-						code += "}\n";
+						// `vertex()` and `light()` are commented out by default
+						// as these are used less often.
+						// Functions are ordered in the vertex-fragment-light order
+						// to follow established conventions.
```

Comment:
I'm not sure the user will need these reminders every time they create a new shader. 

I would remove these comments. 

I would also have the functions un-commented by default. If they are empty, they don't do anything, so commenting them out doesn't change the generated shader at all.